### PR TITLE
WARNING about backups: service instance backup vs. DR backup

### PR DESCRIPTION
**Pull request by GETandSELECT**
please check English

### CODE REVIEW

File: mariadb.html.md.erb [hunk 1]
```diff
@@ -112,12 +112,12 @@ See [Migrate from legacy MariaDB to MariaDB Ent](./migrate-mariadb-to-mariadbent
 
 
 ## <a id='backup'></a> Backup
-Within our MariaDB service you can start a backup and also restore these afterwards. 
-For more information, check the [service backup](../devguide/services/backups.html) guide.
+Within your MariaDB service instance you can start a backup and also restore these afterwards.
+For more information about this Developer Portal feature, check the [service backup](../devguide/services/backups.html) guide.
```
Comment (MatthiasWinzeler) [inline mariadb.html.md.erb]: It's not a developer portal feature, it can also by done via CLI with the swisscom cli plugin :)

Comment (GETandSELECT) [reply mariadb.html.md.erb]: fixed. Thanks.

File: mariadb.html.md.erb [hunk 2]
```diff
@@ -112,12 +112,12 @@ See [Migrate from legacy MariaDB to MariaDB Ent](./migrate-mariadb-to-mariadbent
...
-Within our MariaDB service you can start a backup and also restore these afterwards. 
-For more information, check the [service backup](../devguide/services/backups.html) guide.
+Within your MariaDB service instance you can start a backup and also restore these afterwards.
+For more information about this feature, check the [service backup](../devguide/services/backups.html) guide.
 
-Some of our users also created apps for a regular/scheduled backup. An example below:
-- https://github.com/theonlyandone/cf-backup-app
+Some of our users also created apps for a regular/scheduled backup. An example [here](https://github.com/theonlyandone/cf-backup-app).
 
+We do twice a day full backup for Disaster recovery, of the hole MariaDB/Galera cluster (not individual databases), with `xtrabackup`. If you accidentally `DROP` a table or other error happens, we won't restore your table. Please make sure you make in Portal a [service backup](../devguide/services/backups.html) before you do changes.
```
Comment (annaronco) [inline mariadb.html.md.erb]: We do full backup for disaster recovery  twice a day. This includes the whole MariaDB/Galera cluster (not individual databases), with `xtrabackup`. If you accidentally `DROP` a table or other error happens, we won't restore it. Please make sure that you enable [service backups](../devguide/services/backups.html) in the portal before you make changes.

File: mariadb.html.md.erb [hunk 3]
```diff
@@ -112,10 +112,10 @@ See [Migrate from legacy MariaDB to MariaDB Ent](./migrate-mariadb-to-mariadbent
 
 
 ## <a id='backup'></a> Backup
-Within your MariaDB service instance you can start a backup and also restore these afterwards.
-For more information about this feature, check the [service backup](../devguide/services/backups.html) guide.
+Within your MariaDB service instance, you can start a backup and also restore these afterwards.
+For more information about this feature, check the [service backup guide](../devguide/services/backups.html).
 
-Some of our users also created apps for a regular/scheduled backup. An example [here](https://github.com/theonlyandone/cf-backup-app).
+Some of our users also created apps for a regular/scheduled backup. See example of the [automatic backups app.](https://github.com/theonlyandone/cf-backup-app).
```
Comment (annaronco) [inline mariadb.html.md.erb]: See the example ...

File: mariadb.html.md.erb [hunk 4]
```diff
@@ -112,10 +112,10 @@ See [Migrate from legacy MariaDB to MariaDB Ent](./migrate-mariadb-to-mariadbent
 
 
 ## <a id='backup'></a> Backup
-Within your MariaDB service instance you can start a backup and also restore these afterwards.
-For more information about this feature, check the [service backup](../devguide/services/backups.html) guide.
+Within your MariaDB service instance, you can start a backup and also restore these afterwards.
```
Comment (annaronco) [inline mariadb.html.md.erb]: restore this afterwards.

**Pull request merged**